scripts/replace_protos.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const request = require('request');
+
+const default_mlmd_version = "0.25.1";
+const mlmd_version = process.argv.slice(2)[0] || default_mlmd_version;
+
+// Step#1 delete two original files
+const metadata_store_file_path = "./proto/ml_metadata/proto/metadata_store.proto";
+const metadata_store_service_file_path = "./proto/ml_metadata/proto/metadata_store_service.proto";
+
+try {
+  if (fs.existsSync(metadata_store_file_path)) {
+    fs.unlinkSync(metadata_store_file_path);
+  }
+
+  if (fs.existsSync(metadata_store_service_file_path)) {
+    fs.unlinkSync(metadata_store_service_file_path);
+  }
+  console.log("Step#1 finished: deleted two original files!");
+} catch(err) {
+  console.error(err)
+}
+
+// Step#2 download two new proto files
+const new_metadata_store_file_path = "https://raw.githubusercontent.com/google/ml-metadata/v" + mlmd_version + "/ml_metadata/proto/metadata_store.proto";
+const new_metadata_store_service_file_path = "https://raw.githubusercontent.com/google/ml-metadata/v" + mlmd_version + "/ml_metadata/proto/metadata_store_service.proto";
+download(
```

Comment:
I can see you have put some efforts implementing these. They are great and does the job!
I don't think we need to change.

Just a note: If I were implementing the script, I would write the script in bash instead, tools like `wget` or `curl` can all download the file in one line. It's possible running a bash script from `package.json/scripts` too.

So far, most of the developing scripts assume developer has access to either mac or linux. It's okay for us to not accommodate for windows.

Reply:
Thanks for the note! Learned a lot.